docs/enhancement-proposals/off-ep-0009.md

```diff
@@ -0,0 +1,88 @@
+
+Users may themselves wish to tinker with options specified in a SMIRNOFF force field, such as not using LJPME even if `periodic_method="Ewald3D"` is specified. There is nothing a force field specification can do to prevent modifications like this, identically to other potentially-disruptive user modifications such as changing the cut-off distance.
+
+This proposal only adds a non-default option and does not make recommendations of which option is best.
+
+In this first iteration, `periodic_method="Ewald3D"` is only compatible with `potential="Lennard-Jones-12-6"`, which is currently the only supported value. Future changes to the `potential` attribute should discuss compatibility with LJPME, if any, including which terms can be ignored in reciprocal space.
+
+## Backward compatibility
+
+This proposal only *adds a new supported value* for one attribute and makes no other changes, so it should be backwards-compatible with all current and compliant implementations. Conversion from version 0.4 should not change the information content of an OFFXML file or in-memory representation.
+
+This proposal bumps the version of the vdW section from 0.4 to 0.5 with only the being that `"Ewald3D"` is a supported value for `periodic_potential`.
+
+## Detailed description
+
+This change adds `"Ewald3D"` as a supported value of the `periodic_method` attribute in the `<vdW>` tag:
+
+```
+* `Ewald3D`: a method like [particle mesh Ewald](https://docs.openmm.org/latest/userguide/theory.html#coulomb-interaction-with-particle-mesh-ewald) should be used. This is only compatible with `potential="Lennard-Jones-12-6"`.
+```
+
+The description is nearly identical to how the `periodic_potential` attribute of the `<Electrostatics>` section is described.
+
+This change corresponds to a bump in the vdW version from 0.4 to 0.5. All other aspects of the vdW section remain unchanged, including the default `"periodic_method="cutoff"`.
+
+## Alternatives
+
+[OFF-EP-0007b](https://github.com/openforcefield/standards/pull/44) includes this change as part of a larger overhaul with long-range dispersion corrections. That was introduced before [OFF-EP-0008](https://github.com/openforcefield/standards/pull/53) split the `method` attribute into `periodic_method` and `nonperiodic_method` attributes and it placed the LJPME option within a new `long_range_treatment` attribute. This might lead to confusing and self-inconsistent combinations of attribute values such as
+
+```XML
+<vdW ... periodic_method="cutoff" nonperiodic_method="no-cutoff" long_range_treatment="Ewald3D-ConductingBoundary" </vdW>
+```
+
+This proposal suggests that `nonperiodic_method` is a more natural place to specify the use of LJPME.
+
+This proposal does not attempt to generally resolve current ambiguities in long-range disperson treatment or how cut-off vdW interactions are handled.
+
+## Discussion
+
```

Comment:
Perhaps we could capture some of our discussion here for future consideration?
For example:
* The implementation of Ewald 3D summation should use the equivalent of conducting periodic boundary conditions
* It can efficiently be implemented via PME (cite Darden introduction paper)
* Implementations may need to use certain approximations or methods to deal with non-geometric combining rules in PME (refs)
* Implementation is generally restricted to 1/r^6 since 1/r^12 is much shorter ranged
* The Ewald3D solution can technically be extended to support any 1/r^{2n} powers, but we are not considering this for now

Reply:
Added these points and some links in cd7571a943146716087b85911d1ea2d09dedc493

Reply:
This is great, thanks!